doc/lsst.pex.config/inspecting-configs.rst

```diff
@@ -20,8 +20,12 @@ The `Config.formatHistory` method displays the history of a given `Field` in a m
 Docstrings
 ----------
 
-Each `Field` instance is required to have a doc string that describes the contents of the field.
-Doc strings can be verbose and should give users of the `Config` object a good understanding of what the field is and how it will be interpreted and used.
-A doc string should also be provided for the class as a whole.
+Each `Field` instance is required to have a ``doc`` attribute that describes the contents of the field.
+These ``doc`` strings should be complete sentences and should give users of the `Config` object a good understanding of what the field is and how it will be interpreted and used.
+Multi-sentence ``doc`` strings should have an explicit newline character (``\n``) after each sentence;
```

Comment:
It's not clear to me why multiple short sentences can't be combined onto one line, even in persisted comments.  Maybe change "after each sentence" to "at appropriate intervals"?

Reply:
My thought was that it's hard to identify what "appropriate intervals" for the `config.save` output are in this doc string, so I just went with "every sentence on its own line". Nobody is testing how `config.save` looks when they're writing configs (and I don't expect anybody to start), so I feel like simpler is better, here. It's a "should", not a "must".

I suppose I could make it something like this, but that seems excessive:

```
Multi-sentence ``doc`` strings should have an explicit newline character (``\n``) after each sentence, or at some other appropriate interval if you check both the ``config.save()`` output and the rendered HTML docs;
```

Reply:
I thought the HTML docs don't matter because all the newlines are removed.

Perhaps we should just auto-wrap in `config.save` and eliminate the whole issue.

Reply:
But, to be clear, I can live with `\n` after every sentence.